sonar-project.properties

```diff
@@ -18,3 +18,5 @@ sonar.test.inclusions=src/**/*.test.js,test/**/*
 
 # Coverage
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
+
+sonar.coverage.exclusions=src/**/routes/sharedFields.js
```

Comment:
Hmmmm I wonder why this hasn't been flagged before. I see on line 11 we've excluded steps files I wonder if we just want to exclude all the routes files? They are essentially just config and we have pretty good browser test coverage of how it all integrates with the HMPO libraries

Think in general routes files can be difficult to test, I've seen it done before with snapshot testing but that would require us moving to Jest

Probably not one for now, want to just incorporate this exclusion into the statement above though?